modules/upgrade.adoc

```diff
@@ -20,19 +20,23 @@ All Kubernetes objects and PVs in each {product-title} cluster are backed up as
 ====
 
 [id="upgrade-automatic_{context}"]
-== Automatic upgrades
+== Recurring upgrades
 
 Upgrades can be scheduled to occur automatically on a day and time specified by the cluster owner or administrator. Upgrades occur on a weekly basis, unless an upgrade is unavailable for that week.
 
+If you select recurring updates for your cluster, you must provide an administrator’s acknowledgment. The {cluster-manager} does not start scheduled updates without receiving an administrator's acknowledgment. For information about administrator acknowledgment, see link:https://docs.openshift.com/dedicated/upgrading/osd-upgrading-cluster-prepare.html#upgrade-49-acknowledgement_osd-updating-cluster-prepare[Administrator acknowledgment when upgrading to OpenShift 4.9].
+
 [NOTE]
 ====
-Automatic upgrade policies are optional and if they are not set, the upgrade policies default to manual.
+Recurring upgrade policies are optional and if they are not set, the upgrade policies default to manual.
```

Comment:
"default to manual" maybe better to "default to individual"?

Reply:
Updated